src/zkevm_specs/evm/execution/dataCopy.py

```diff
@@ -0,0 +1,85 @@
+from typing import Optional, Sequence, Tuple, Union, List
+
+from ..instruction import Instruction, Transition
+from ..table import (
+    AccountFieldTag,
+    BlockContextFieldTag,
+    BytecodeFieldTag,
+    CallContextFieldTag,
+    FixedTableRow,
+    RWTableRow,
+    Tables,
+    FixedTableTag,
+    TxContextFieldTag,
+    RW,
+    RWTableTag,
+    TxLogFieldTag,
+    TxReceiptFieldTag,
+    CopyDataTypeTag,
+)
+from ...util import FQ, N_BYTES_MEMORY_ADDRESS, N_BYTES_MEMORY_SIZE, IdentityPerWordGas
+
+
+def dataCopy(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
```

Comment:
Since the step of precompile doesn't have an opcode for it, so this shouldn't be useful here.

I think what we could do is instead in `CALL` step doing lookup  `next.execution_state -> callee_address` to make sure the state transition is reasonable (need to add another `PrecompileAddress` tag in `fixed_table`).
```suggestion
```